packages/apps-engine/package.json

```diff
@@ -0,0 +1,132 @@
+{
+    "name": "@rocket.chat/apps-engine",
+    "version": "1.46.0-alpha",
+    "description": "The engine code for the Rocket.Chat Apps which manages, runs, translates, coordinates and all of that.",
+    "main": "index",
+    "typings": "index",
+    "scripts": {
+        "start": "run-s .:build:clean .:build:watch",
+        "testunit": "run-p .:test:node .:test:deno",
+        ".:test:node": "NODE_ENV=test ts-node ./tests/runner.ts",
+        ".:test:deno": "cd deno-runtime && deno task test",
+        "lint": "run-p .:lint:eslint .:lint:deno",
+        ".:lint:eslint": "eslint .",
+        ".:lint:deno": "deno lint --ignore=deno-runtime/.deno deno-runtime/",
+        "fix-lint": "eslint . --fix",
+        "build": "run-s .:build:clean .:build:default",
+        ".:build:clean": "rimraf client definition server",
+        ".:build:default": "tsc -p tsconfig.json",
+        ".:build:watch": "yarn .:build:default --watch",
+        "typecheck": "tsc -p tsconfig.json --noEmit",
+        "bundle": "node scripts/bundle.js",
+        "gen-doc": "typedoc",
+        "prepack": "yarn bundle",
+        "postinstall": "node scripts/postinstall.js"
```

Comment:
what if you move this to the "build" script? this way we can avoid having to install `deno` in ALL Dockerfiles since the majority of them just uses apps-engine for its types..

Reply:
That's a good point, I'll take a look